pkg/ddc/jindo/const.go

```diff
@@ -1,3 +1,16 @@
+/*
+Copyright 2022 The Fluid Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OD ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the Lcense.
```

Comment:
Please follow the license header [here](https://github.com/fluid-cloudnative/fluid/blob/master/pkg/common/constants_test.go#L10-L14) to break the lines